doc/user-guide/tcs-user-guide-generic.rst

```diff
@@ -342,38 +350,76 @@ In :numref:`fig-offset-xy-example` we show an example of the result of using :py
+Basically, these offsets will accumulate with one another, e.g.;
 
 .. code:: python
 
     await tcs.offset_azel(az=10, el=0)
     await tcs.offset_azel(az=0, el=10)
 
-Is equivalent to a 10 arcsec offset in elevation only;
+Will result in an offset of 10 arcsec in azimuth (from the first command) **and** elevation (from the second).
+Relative offsets are probably the most common and general kind of offsets used and are employed on most use cases.
+
+Non-relative offsets are performed with respect to the pointing origin.
+For instance, the following sequence of commands;
 
 .. code:: python
 
-    await tcs.offset_azel(az=0, el=10)
+    await tcs.offset_azel(az=10, el=0, relative=False)
+    await tcs.offset_azel(az=0, el=10, relative=False)
+
+Is equivalent to a 10 arcsec offset in elevation only and is equivalent to performing the second command alone, e.g.;
 
-The reason is that the second offset command will override the first command.
+.. code:: python
+
+    await tcs.offset_azel(az=0, el=10, relative=False)
 
-On the other hand, relative offsets will accumulate with one another, e.g.;
+Non-relative offsets are useful when performing operations that may use the original position as a reference.
+For instance, when building grids of observations one may want to use non-relative offsets.
+The following command sequence will perform a square grid on the sky (see :numref:`fig-offset-azel-grid`);
 
 .. code:: python
 
-    await tcs.offset_azel(az=10, el=0, relative=True)
-    await tcs.offset_azel(az=0, el=10, relative=True)
+    grid_az = [-10, +10, +10, -10]
+    grid_el = [-10, -10, +10, +10]
+    for az,el in zip(grid_az, grid_el):
+      await tcs.offset_azel(az=az, el=el, relative=False)
+      # Do something here
+      ....
+
+    # Restore the original telescope position.
+    await tcs.offset_azel(az=0., el=0., relative=False)
+
+The same pattern using a relative offset would look like this:
+
+.. code:: python
 
-Will result in an offset of 10 arcsec in azimuth **and** elevation.
+    grid_az = [-10, 0, 20, 0]
+    grid_el = [-10, +20, 0, -20]
+    for az,el in zip(grid_az, grid_el):
+      await tcs.offset_azel(az=az, el=el)
+      # Do something here
+      ....
+
+    # Restore the original telescope position.
+    await tcs.offset_azel(az=10., el=-10.)
+
+As you can see, not only defining the grid is easier but also, if an error occur during the ``for`` loop, it is considerably simpler to restore the telescope to its original position when using the non-relative offset.
+
+.. figure:: /_static/offset_azel_grid.png
+   :name: fig-offset-azel-grid
+   :target: ../_images/offset_azel_grid.png
+   :alt: offset xy example
+
+   Same offset pattern executed with non-relative (left hand side) and relative (right hand side) offsets.
+   The numbers on the images represent the input to the offset command on both scenarios.
 
 Relative and absolute offsets can also be combined.
```

Comment:
you say "absolute"
I suggest doing a search for all uses of absolute.... there are probably others.